receiver/bigipreceiver/testdata/apiresponses/get_pool_members_stats_response_1.json

```diff
@@ -0,0 +1,107 @@
+                    "connq.ageMax": {
+                        "value": 0
+                    },
+                    "connq.depth": {
+                        "value": 0
+                    },
+                    "connq.serviced": {
+                        "value": 0
+                    },
+                    "curSessions": {
+                        "value": 0
+                    },
+                    "monitorRule": {
+                        "description": "/Common/http (pool monitor)"
+                    },
+                    "monitorStatus": {
+                        "description": "down"
+                    },
+                    "mr.msgIn": {
+                        "value": 0
+                    },
+                    "mr.msgOut": {
+                        "value": 0
+                    },
+                    "mr.reqIn": {
+                        "value": 0
+                    },
+                    "mr.reqOut": {
+                        "value": 0
+                    },
+                    "mr.respIn": {
+                        "value": 0
+                    },
+                    "mr.respOut": {
+                        "value": 0
+                    },
+                    "nodeName": {
+                        "description": "/Common/dev"
+                    },
+                    "poolName": {
+                        "description": "/Common/dev"
+                    },
+                    "port": {
+                        "value": 80
+                    },
+                    "serverside.bitsIn": {
+                        "value": 0
+                    },
+                    "serverside.bitsOut": {
+                        "value": 0
+                    },
+                    "serverside.curConns": {
+                        "value": 0
+                    },
+                    "serverside.maxConns": {
+                        "value": 0
+                    },
+                    "serverside.pktsIn": {
+                        "value": 0
+                    },
+                    "serverside.pktsOut": {
+                        "value": 0
+                    },
+                    "serverside.totConns": {
+                        "value": 0
+                    },
+                    "sessionStatus": {
+                        "description": "enabled"
+                    },
+                    "status.availabilityState": {
+                        "description": "offline"
+                    },
+                    "status.enabledState": {
+                        "description": "enabled"
+                    },
+                    "status.statusReason": {
+                        "description": "/Common/http: No successful responses received before deadline. @2022/04/29 08:33:56. "
+                    },
+                    "totRequests": {
+                        "value": 0
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
newline (probably should be on all these API responses).

Reply:
@cpheps changed